register/urls.py

```diff
@@ -1,6 +1,6 @@
-from django.urls import path
-from . import views
-
-app_name = "register"
-
-urlpatterns = [path("<str:user_type>/", views.register_user, name="register_user")]
+from django.urls import path
```

Comment:
was there any change in this file? the code seems to be the same

Reply:
it must have been how I merged it, but no changes were made.